client/src/accounts/authentication/register.hook.test.tsx

```diff
@@ -88,12 +84,12 @@ describe('useRegisterHook', () => {
     act(() => {
       result.current.mutate(registerForm);
     });
+
     await waitFor(() => expect(result.current.isError).toBe(true));
     expect(result.current.error).toEqual({ message: expectedMessage });
   });
 
-  it('should return if registration was successful', async () => {
-    // happy path
+  it('should show success code if registration was successful', async () => {
```

Comment:
It doesn't show the success code, it returns it as part of the response, nothing in a hook test is shown.